docs/book/getting-started/zenml-pro/self-hosted.md

```diff
@@ -0,0 +1,1118 @@
+---
+description: >
+  Guide for installing ZenML Pro self-hosted in a Kubernetes cluster.
+---
+
+# ZenML Pro Self-Hosted Deployment
+
+This page provides instructions for installing ZenML Pro - the ZenML Pro Control Plane and one or more ZenML Pro Tenant servers - on-premise in a Kubernetes cluster.
+
+## Overview
+
+ZenML Pro can be installed as an self-hosted deployment. You'll need access to the ZenML Pro container images and helm chart and you'll have to provide your own infrastructure: a Kubernetes cluster, a database server and a few other common prerequisites usually needed to expose Kubernetes services via HTTPs - a load balancer, an Ingress controller, HTTPs certificate(s) and DNS rule(s).
```

Comment:
Now that the helm chart is public, do we still need to mention it here?

Reply:
no, we don't. Fixed it